### PR TITLE
Update DOI

### PR DESCRIPTION
**Pull request by Peiffap**

### CODE REVIEW

**Comment by matbesancon** [comment]
thanks, could you update it in CITATION.bib too?

**Comment by matbesancon** [comment]
@Peiffap I found an easier way on Zenodo, we can cite 10.5281/zenodo.2647458 which is the permanent DOI always resolving to the latest version (sorry for the multiple changes)

**Comment by Peiffap** [comment]
> @Peiffap I found an easier way on Zenodo, we can cite 10.5281/zenodo.2647458 which is the permanent DOI always resolving to the latest version (sorry for the multiple changes)

Ah, yes that's a much smarter way to do it; however, in the CITATION file, the month (and year, eventually), will still have to be update manually I'm afraid.

File: CITATION.bib [hunk 1]
```diff
@@ -16,6 +16,6 @@ @misc{Distributions.jl-2019
   title        = {{JuliaStats/Distributions.jl: a Julia package for probability distributions and associated functions}},
   month        = apr,
   year         = 2019,
-  doi          = {10.5281/zenodo.2647520},
-  url          = {https://doi.org/10.5281/zenodo.2647520}
+  doi          = {10.5281/zenodo.2650434},
```
Comment (Peiffap) [inline CITATION.bib]:
```suggestion
  doi          = {10.5281/zenodo.2647458},
```

File: CITATION.bib [hunk 2]
```diff
@@ -16,6 +16,6 @@ @misc{Distributions.jl-2019
...
+  url          = {https://doi.org/10.5281/zenodo.2650434}
```
Comment (Peiffap) [inline CITATION.bib]:
```suggestion
  url          = {https://doi.org/10.5281/zenodo.2647458}
```

File: README.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ Distributions.jl
 
 [![Travis status](https://travis-ci.org/JuliaStats/Distributions.jl.svg?branch=master)](https://travis-ci.org/JuliaStats/Distributions.jl)
 [![Appveyor status](https://ci.appveyor.com/api/projects/status/xqm07gyruflhnha7/branch/master?svg=true)](https://ci.appveyor.com/project/simonbyrne/distributions-jl/branch/master)
-[![](https://zenodo.org/badge/DOI/10.5281/zenodo.2647520.svg)](https://zenodo.org/record/2647520)
+[![](https://zenodo.org/badge/DOI/10.5281/zenodo.2650434.svg)](https://zenodo.org/record/2650434)
```
Comment (Peiffap) [inline README.md]:
```suggestion
[![](https://zenodo.org/badge/DOI/10.5281/zenodo.2647458.svg)](https://zenodo.org/record/2647458)
```

File: CITATION.bib [hunk 4]
```diff
@@ -16,6 +16,6 @@ @misc{Distributions.jl-2019
   title        = {{JuliaStats/Distributions.jl: a Julia package for probability distributions and associated functions}},
   month        = apr,
```
Comment (Peiffap) [inline CITATION.bib]:
```suggestion
  month        = may,
```

**Comment by matbesancon** [comment]
well updating the date is fine, we'll manage and worst case the bibtex entry does not become incorrect

**Comment by matbesancon** [comment]
Thanks :)

**Comment by Nosferican** [comment]
Might not be needed in this case, but a good practice is to have authors listed as

Surname, FirstName MiddleName and Surname, FirstName MiddleName and...

Bib understands the format and allows for other naming conventions besides single surnames (e.g., English) such as Latin, Asian, Slavic, etc.

**Comment by matbesancon** [comment]
I believe this is more of a Zenodo issue in that case. Formatting names from github info to a BibTeX must be a nightmare for them

**Comment by Peiffap** [comment]
Does/did Zenodo automatically generate the CITATION.bib file ?

**Comment by Nosferican** [comment]
It does. The functionality is quite primitive, but it gets the job done in most cases. Using other tools like Zotero and the DOI will prompt the tool to query Zenodo for the data.

**Pull request merged**